pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -272,6 +272,9 @@ def _attach_model_callbacks(self) -> None:
         model_callbacks = self.trainer._call_lightning_module_hook("configure_callbacks")
         if not model_callbacks:
             return
+
+        model_callbacks = [model_callbacks] if not isinstance(model_callbacks, Sequence) else model_callbacks
+        model_callbacks = list(model_callbacks)
```

Comment:
n00b q: is this list conversion strictly needed? if someone passed a tuple/set of callbacks, is that an issue?

Reply:
```suggestion
```

Reply:
true, not required.